packages/opentelemetry/src/spanExporter.ts

```diff
@@ -281,6 +281,7 @@ function createAndFinishSpanForOtelSpan(node: SpanNode, spans: SpanJSON[], remai
     op,
     origin,
     _metrics_summary: getMetricSummaryJsonForSpan(span as unknown as Span),
+    measurements: timedEventsToMeasurements(span.events),
```

Comment:
do we actually need/want this here? 🤔

Reply:
I guess it's fine, lets leave it in!

Reply:
If there are not measurements, the key gets dropped anyway :)